### DISCUSSION TITLE
Does stern follow semver?

### DISCUSSION THREAD

**Discussion by bartmanus** [stern/stern#133] [Q&A] [answered]
Hi. Does this stern project follow semver?

I am specifically interested into whether there is any principle about not having breaking changes inside a major version.

Seeing [release v1.15.0](https://github.com/stern/stern/releases/tag/v1.15.0) it seems that the addition of TZ support is not actually breaking the CLI, however the output may be different.

Thanks.

**Comment by superbrothers** [comment, marked as answer]
stern chooses a method that is commonly referred to as semantic versioning. However, it does not assert that it follows it "completely".

It is true that the default behavior of the `--timestamps` flag was changed in v1.15.0, but we also provided a way to keep the previous default, so we made it a minor version release.
